README.md

```diff
@@ -331,6 +332,32 @@ references:
 
 ![](img/manual-reference.png)
 
+### Unicode Considerations
+
+If you build for LaTex output, and you have Unicode characters in any of your paper's source code, you may have problems.  First of all, the default pdf engine simply does not support Unicode characters at all.  You can add `--pdf-engine=xelatex` to the call to `pandoc` in the Makefile to use xelatex as your engine instead.  That gives you access to some font selections for different parts of your paper (see the Fonts section of the Pandoc manual at https://pandoc.org/MANUAL.html).  The one of these that controls your source code fonts is `monofont`.  You can add a line with you `monofont` choice to your YAML metadata block:
```

Comment:
- "LaTex" -> "LaTeX"
- Let's use the direct link https://pandoc.org/MANUAL.html#fonts
- "The one of these" -> "The option"
- "with you `monofont`" -> "with your `monofont`"

I'm a bit confused. Is `monofont: DejaVu Sans Mono` related to Unicode? or is it just choosing a different font for code blocks?

Reply:
`monofont` just lets you pick a different font for mono-spaced test, but the DejaVu fonts provide pretty extensive Unicode coverage (though they don't cover everything).

Reply:
I made those changes, and also made is clear why I picked DejaVus Sans Mono.